package.json

```diff
@@ -2,10 +2,9 @@
   "name": "msw",
   "version": "2.7.6",
   "description": "Seamless REST/GraphQL API mocking library for browser and Node.js.",
-  "type": "commonjs",
-  "main": "./lib/core/index.js",
-  "module": "./lib/core/index.mjs",
-  "types": "./lib/core/index.d.ts",
+  "type": "module",
```

Comment:
This is a breaking change even if we define `require` fallbacks. Bear in mind, MSW is also used with really old tools that won't even look at our export conditions. To them, `require('msw')` that has `type: module` will be a runtime exception.